scripts/cms-any/dispatch-ci.php

```diff
@@ -23,6 +24,26 @@
   schedule:
     - cron: '$cron'
   workflow_dispatch:
+    inputs:
+      major_type:
+        description: 'Major branch type'
+        required: true
+        type: choice
+        options:
+          - 'dynamic'
+          - 'current'
+          - 'next'
+          - 'previous'
+        default: 'dynamic'
+      minor_type:
+        description: 'Minor branch type'
+        required: true
+        type: choice
+        options:
+          - 'dynamic'
+          - 'next-minor'
+          - 'next-patch'
+        default: 'dynamic'
```

Comment:
What are these new inputs for? Please add details to the PR description.

Reply:
NEvermind I found it in the description for https://github.com/silverstripe/gha-dispatch-ci/pull/27

> I've added support in the module-standariser PR linked on the parent issue to trigger dispatch-ci with specific input options to make testing this much easier.

I'd prefer for this to remain simple rather than adding extra complexity - but I feel like I'd get pushback on that. Please at least add comments in here so when we look at this it's clear:
1. what these are even here for (i.e. testing only and should not generally be used outside of testing purposes)
2. what each option actually does

Reply:
I've done 1. and for 2. I said refer to the repo